package.json

```diff
@@ -23,7 +23,7 @@
     "publish:canary": "lerna publish --force-publish --canary --include-merged-tags --preid canary --dist-tag canary --yes --loglevel verbose",
     "release": "node ./tasks/release/cli.mjs",
     "test": "lerna run test --stream -- --colors --maxWorkers=4",
-    "test-release": "NODE_OPTIONS=--experimental-vm-modules ./node_modules/.bin/jest --config ./tasks/release/jest.config.mjs"
+    "test:release": "NODE_OPTIONS=--experimental-vm-modules ./node_modules/.bin/jest --config ./tasks/release/jest.config.mjs"
```

Comment:
I'm not sure why I named it `test-release` when the convention for scoping scripts is to use a colon.

Also, I don't have any plans of adding this to CI, but I do run the tests locally while developing.

Reply:
Maybe this should be `test:release-script` as it's not testing the actual release (or just something else more descriptive).